src/bundle.rs

```diff
@@ -42,9 +56,22 @@ pub unsafe trait DynamicBundle {
 /// A statically typed collection of components
 ///
 /// Bundles composed of exactly the same types are semantically equivalent, regardless of order. The
-/// interface of this trait is a private implementation detail.
+/// interface of this trait, except `has_static`, is a private implementation detail.
 #[allow(clippy::missing_safety_doc)]
 pub unsafe trait Bundle: DynamicBundle {
+    /// Checks if the Bundle contains the given `T`:
+    ///
+    /// ```
+    /// fn test_bundle<T: hecs::Bundle>(input: T) {
+    ///     assert!(T::has_static::<i32>());
+    ///     assert!(T::has_static::<f32>());
+    ///     assert!(!T::has_static::<usize>());
+    /// }
+    /// ```
+    fn has_static<T: Component>() -> bool {
```

Comment:
What's the use case for this?

Reply:
in my own case, I can use this directly rather than `has`, but probably it ends up all the same after the optimizer. I haven't checked yet

Reply:
The static variants of the other methods are motivated by cases where there's no `self` to call on (e.g. `remove`). Maybe that makes sense here too, but it bears some consideration if it's not part of your use case.

Could we name this `has` as well, or would that have worse ergonomics in practice?

Reply:
Which doesn't describe any of my use cases. I'd be for removing this method (which incidentally, yeah, has bad ergonomics with the name `has`) until such time as its needed by anyone then